platform/mds/mds-web/src/main/resources/webapp/js/directives.js

```diff
@@ -502,6 +502,9 @@
             link: function (scope, element, attr, ngModel) {
                 var isReadOnly = scope.$eval(attr.ngReadonly);
                 if(!isReadOnly) {
+                    angular.element(element).click( function() {
```

Comment:
Is this a good solution? We switch classes onclick? Isn't there a way of doing this without js? I would imagine the button isn't supposed to be greyed out by default. 

The old class returns after I click the 'Now' btw.